vm/systemSmartContracts/staking.go

```diff
@@ -1185,7 +1211,7 @@ func (r *stakingSC) getStatus(args *vmcommon.ContractCallInput) vmcommon.ReturnC
 		return returnCode
 	}
 
-	if stakedData.Jailed {
+	if stakedData.Jailed || r.eei.CanUnJail(args.Arguments[0]) {
```

Comment:
When can the validator not be jailed, but CanUnJail?

Reply:
because there might be slight differences between stakingSC and validator statistics. at end of epoch not all with BAD rating are jailed - those still selected (as they are needed) might turn on their PC and have their rating back to normal values again